deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
@sosiouxme @liggitt any objections to letting the deployer be able to list and label (patch) nodes in the cluster for scaling down/up the fluentd pods as part of upgrading?